### DISCUSSION TITLE
A way to finance propositions

### DISCUSSION THREAD

**Discussion by Ipickedausername** [opengisch/QField#3738] [Ideas]
Hello, 

Many propositions are made to improve QFIELD. But without funding, difficult to develop. 

Do you think it could be possible to have a dedicated page where the enhancement propositions are listed. 
A developper could then define the time and cost of developping the proposition, a short explaination on how he will proceed and what to expect.

Then, a link is provided to participate fund the proposition. When the funding goal is achieved, the propisition can be developped (without any schedule).

It would give more visibility to people who want to participate, we can also have a specific idea on what we have funded. 

If something is missing on the app and a proposal already exists, we can easily see if the funding requirement have been reached or if we can participate. 

Do you think it could help?

**Comment by mbernasocchi** [comment]
@Ipickedausername Thanks a lot for the idea; sounds very interesting. We'll check its feasibility internally and will get back to you.
Cheers Marco

**Reply by mbernasocchi** [reply]
ideas.qfield.org is a first step in this direction.
